doc/changes/changes_1.1.1.md

```diff
@@ -0,0 +1,25 @@
+# Virtual Schema for document data in files on Azure Blob Storage 1.1.1, released 2022-09-28
+
+Code name: CVE Warnings
+
+## Summary
+
+Fix CVE Warnings + Dependency Updates
+
```

Comment:
No ticket number?

Reply:
It's in the PR, but I'm going to add it.